client/modules/IDE/reducers/files.js

```diff
@@ -30,6 +31,15 @@ const defaultCSS =
   margin: 0;
   padding: 0;
 }
+
+#accessible-outputs {
```

Comment:
i think this is the last thing, but this CSS should also get injected by the library!